linux_os/guide/system/network/network-nftables/service_nftables_disabled/rule.yml

```diff
@@ -39,7 +39,7 @@ ocil: |-
 
 fixtext: '{{{ fixtext_service_disabled("nftables") }}}'
 
-platform: machine
+platform: machine and package[nftables] and package[firewalld]
```

Comment:
I think it is good to include `and package[nftables]` but it not necessary to include `and package[firewalld]`

Reply:
Same motivation as for default_deny_policy rule

Reply:
Ideally we should also have a `service_enabled[firewalld]` here.